azure-pipelines.yml

```diff
@@ -6,7 +6,7 @@ variables:
   NugetPackageVersion: '$(CurrentSemanticVersion)'
   MONO_VERSION: 6_4_0
   XCODE_VERSION: 11.1
-  NETCORE_VERSION: '3.0.x'
+  NETCORE_VERSION: '3.1.x'
   NETCORE_TEST_VERSION: '2.2.x'
```

Comment:
Are we still on netcore 2.1 for the tests?

Reply:
yeah that's another issue we can address separately.